packages/calypso-e2e/src/lib/components/editor-settings-sidebar-component.ts

```diff
@@ -13,25 +19,23 @@ interface Schedule {
 	meridian: 'am' | 'pm';
 }
 
-const sidebarParentSelector = '[aria-label="Editor settings"]';
+const panel = '[aria-label="Editor settings"]';
 
 const selectors = {
+	// Close button for mobile
+	mobileCloseSidebarButton: `${ panel } [aria-label="Close settings"]:visible`,
+
 	// Tab
-	tabButton: ( tabName: EditorSidebarTab ) =>
-		`${ sidebarParentSelector } button:has-text("${ tabName }")`,
+	tabButton: ( tabName: EditorSidebarTab ) => `${ panel } button[data-label="${ tabName }"]`,
 	activeTabButton: ( tabName: EditorSidebarTab ) =>
-		`${ sidebarParentSelector } button.is-active:has-text("${ tabName }")`,
+		`${ panel } button.is-active:has-text("${ tabName }")`,
 
-	// Sections
-	sectionToggle: ( sectionName: EditorSidebarSection ) =>
-		`${ sidebarParentSelector } .components-panel__body-toggle:has-text("${ sectionName }")`,
-	expandedSection: ( sectionName: EditorSidebarSection ) =>
-		`${ sidebarParentSelector } .is-opened .components-panel__body-toggle:has-text("${ sectionName }")`,
-	revisionsToggle: `${ sidebarParentSelector } .components-panel__body:has-text("Revisions")`,
-	lastSection: `${ sidebarParentSelector } .components-panel__body >> nth=-1`,
+	// General section-related
+	section: ( name: ArticleSections ) => `${ panel } button:has-text("${ name }")`,
+	sectionHeader: `${ panel } .components-panel__body`,
```

Comment:
I believe this is just for the revisions link, right? Should we just collapse this with the section above by keying off of the `.components-button` class? It feels a little weird having these two selectors as they seem at surface level to be referencing the same thing, and I could see folks being initially confused which they should use.

Reply:
Revisions is a bit different in that it's an `<a>` tag.

I've changed this selector to point specifically to the revisions button instead.

Reply:
So I remember why this selector was here, it was to handle the scrolling of the sidebar to the end.

Reply:
I've changed the `scrollToBottomOfSidebar` method to scroll to the `Discussion` panel, which is the last item on the page.